vue/src/core/components/layout/AuthLayout/AuthLayout.vue

```diff
@@ -67,7 +67,7 @@ export default {
   }
 
   & /deep/ .btn-action {
-    border-radius: 1.5rem;
+    border-radius: 0;
```

Comment:
@ll-bat This is not necessary. You just need to remove `.btn-action` class from buttons